Modules/Filtering/FFT/src/itkFFTWGlobalConfiguration.cxx

```diff
@@ -157,15 +157,9 @@ FFTWGlobalConfiguration::GetInstance()
       m_PimplGlobals->m_Instance = Self::New();
       if (!m_PimplGlobals->m_Instance)
       {
-        std::ostringstream message;
-        message << "itk::ERROR: "
-                << "FFTWGlobalConfiguration"
-                << " Valid FFTWGlobalConfiguration instance not created";
-        itk::ExceptionObject e_(__FILE__, __LINE__, message.str().c_str(), ITK_LOCATION);
-        throw e_; /* Explicit naming to work around Intel compiler bug.  */
+        itkExceptionMacro("Valid FFTWGlobalConfiguration instance not created");
```

Comment:
This is a static method. The `itkExceptionMacro` using `this`, so this should not work if FFTW is enable in CI.

Reply:
Oops, I see now, after having enabled `ITK_USE_FFTWD` my local VS2022 build says:

> itkFFTWGlobalConfiguration.cxx(160,9): error C2355: 'this': can only be referenced inside non-static member functions or non-static data member initializers

Thanks @blowekamp Should be fixed with my next force-push!